arches/app/media/js/views/components/datatypes/resource-instance.js

```diff
@@ -68,6 +68,9 @@ define([
                 });
                 graph.ontologyProperty = ko.observable(ko.unwrap(graph.ontologyProperty));
                 graph.inverseOntologyProperty = ko.observable(ko.unwrap(graph.inverseOntologyProperty));
+                graph.relationshipConcept = ko.observable(ko.unwrap(graph.relationshipConcept) || 'ac41d9be-79db-4256-b368-2f4559cfbe55');
```

Comment:
I think it would be nice to declare default relationship uuid up at the top and use the variable here so that it is clear to developers that `ac41d9be-79db-4256-b368-2f4559cfbe55` is the `is related to` concept value.